android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java

```diff
@@ -3187,6 +3195,33 @@ public int getMyLocationTrackingMode() {
         return mUserLocationView.getMyLocationTrackingMode();
     }
 
+    /**
+     * Set the current my bearing tracking mode.
+     * Tracking my bearing disables gestures and shows the direction the user is heading.
```

Comment:
Can we add a comment about the behaviour when location tracking is off? That this value changes the bearing arrow behaviour?